pages/electronics-gifting-cities.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+
+import * as laptopCitiesData from '../data/laptop-cities.json';
+
+import {
+  GoogleMap,
+  InfoWindow,
+  Marker,
+  withGoogleMap,
+  withScriptjs,
+} from 'react-google-maps';
+
+function Map() {
+  const [selectedcity, setSelectedcity] = useState(null);
+
+  return (
+    <GoogleMap defaultCenter={{ lat: 34.5794, lng: -118.1164 }} defaultZoom={4}>
```

Comment:
maybe we could keep it in some const with name of the place?

Reply:
Good feedback @marekrozmus. 

In this project, we're aiming to have all code follow the [ESLint no-magic-numbers rule](https://eslint.org/docs/rules/no-magic-numbers). 

Try this @ozer619 to make your code more explicit:

1. Define a `const` with a name like `MAP_CENTER_MHF_OFFICE` per [ES6 naming conventions].(https://stackoverflow.com/questions/40291766/naming-convention-for-const-object-keys-in-es6/40291962)
2. Add a comment like `// The MHF office at 1543 E Palmdale Blvd. Suite E, Palmdale, CA 93550` to help other developers understand what the `lat`, `lng` numbers are referring to.